vncviewer/UserDialog.cxx

```diff
@@ -62,34 +63,54 @@ static void button_cb(Fl_Widget *widget, void *val) {
   widget->window()->hide();
 }
 
+static void keepPassword(Fl_Widget *widget, void *data) {
+  UserDialog *dialog = (UserDialog*) data;
+  Fl_Check_Button* flcb = (Fl_Check_Button*)widget;
+  dialog->setKeepPassword(flcb->value());
+}
```

Comment:
You could probably simplify this by simply checking the check box once the dialog is closed. We don't really need to know it's value until afterwards anyway.